frontends/ol-components/src/components/LearningResourceExpanded/InfoSectionV2.tsx

```diff
@@ -376,6 +426,77 @@ const InfoItem = ({ label, Icon, value }: InfoItemProps) => {
   )
 }
 
+const DifferingRunsTable: React.FC<{ resource: LearningResource }> = ({
+  resource,
+}) => {
+  if (!resource.runs) {
+    return null
+  }
+  if (resource.runs.length === 1) {
+    return null
+  }
+  const asTaughtIn = resource ? showStartAnytime(resource) : false
+  const prices = []
+  const deliveryMethods = []
+  const locations = []
+  for (const run of resource.runs) {
+    if (run.prices) {
+      prices.push(run.prices)
+    }
+    if (run.delivery) {
+      deliveryMethods.push(run.delivery)
+    }
+    if (run.location) {
+      locations.push(run.location)
+    }
+  }
+  const distinctPrices = [...new Set(prices.flat())]
+  const distinctDeliveryMethods = [
+    ...new Set(deliveryMethods.flat().map((dm) => dm?.code)),
+  ]
+  const distinctLocations = [...new Set(locations.flat().map((l) => l))]
+  if (
+    distinctPrices.length > 1 ||
+    distinctDeliveryMethods.length > 1 ||
+    distinctLocations.length > 1
+  ) {
+    return (
+      <DifferingRuns data-testid="differing-runs-table">
+        <DifferingRunHeader>
+          <DifferingRunLabel>Date</DifferingRunLabel>
+          <DifferingRunLabel>Price</DifferingRunLabel>
+          <DifferingRunLabel>Format</DifferingRunLabel>
+        </DifferingRunHeader>
+        {resource.runs.map((run, index) => (
+          <DifferingRun key={index}>
+            <DifferingRunData>
+              {formatRunDate(run, asTaughtIn)}
+            </DifferingRunData>
+            {run.prices && (
+              <DifferingRunData>
+                <span>{run.prices.map((p) => `$${p}`).join(", ")}</span>
```

Comment:
In other places we display exact dollar amounts without decimals. Check `getDisplayPrecision()` in ol-utilities/src/learning-resources/pricing.ts